.github/workflows/deploy-previews.yml

```
@@ -1065,6 +1071,74 @@
+            --timeout=60 \
+            --concurrency=10 \
+            --image=${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/pr-previews/superset-pr-previews:${{ github.sha }} \
+            --port=8088 \
+            --cpu=1000m \
+            --memory=1024Mi \
+            --update-secrets=/etc/env/.env=sta-env-superset:latest \
+            --command="/bin/sh","-c","cat /etc/env/.env >> /app/.env; /app/docker-entrypoint
+
+  superset-pr-comment:
+    name: superset-preview-link-comment
+    if: needs.check.outputs.superset == 'true'
```

actionlint (1.7.4)

property "check" is not defined in object type {superset: {outputs: {url: string}; result: string}}

(expression)